example/hello-osiris.yaml

```diff
@@ -45,6 +45,7 @@ metadata:
   annotations:
     osiris.deislabs.io/enabled: "true"
     osiris.deislabs.io/minReplicas: "1"
+    osiris.deislabs.io/ignoredPaths: "/first/path,/second-path"
```

Comment:
Just checking... does k8s actually allow a comma here?

Reply:
yes, and some people/tools even store JSON in annotations

Reply:
I could have sworn there were some limitations on characters used in annotation values, but double-checking [here](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set) myself, I only see limitations mentioned on the characters used in the keys. 👍